http.kafka.cqrs/src/main/java/io/aklivity/example/cqrs/processor/CommandProcessor.java

```diff
@@ -0,0 +1,66 @@
+package io.aklivity.example.cqrs.processor;
+
+import java.util.Arrays;
+
+import org.apache.kafka.common.header.Header;
+import org.apache.kafka.common.header.Headers;
+import org.apache.kafka.common.header.internals.RecordHeaders;
+import org.apache.kafka.streams.processor.api.Processor;
+import org.apache.kafka.streams.processor.api.ProcessorContext;
+import org.apache.kafka.streams.processor.api.ProcessorSupplier;
+import org.apache.kafka.streams.processor.api.Record;
+import org.apache.kafka.streams.state.KeyValueStore;
+
+import io.aklivity.example.cqrs.model.Command;
+
+public class CommandProcessor implements ProcessorSupplier<String, Command, String, Command>
+{
+    private final String commandStore;
+    private String successfulCommandProcessor;
+    private String failedCommandProcessor;
+    private KeyValueStore<String, byte[]> commandStateStore;
+
+    public CommandProcessor(String commandStore, String successfulCommandProcessor, String failedCommandProcessor)
+    {
+        this.commandStore = commandStore;
+        this.successfulCommandProcessor = successfulCommandProcessor;
+        this.failedCommandProcessor = failedCommandProcessor;
+    }
+
+    @Override
+    public Processor<String, Command, String, Command> get()
+    {
+        return new Processor<>()
+        {
+            ProcessorContext context;
+
+            @Override
+            public void init(final ProcessorContext context)
+            {
+                this.context = context;
+                commandStateStore = (KeyValueStore) context.getStateStore(commandStore);
```

Comment:
Since the intention of using a supplier is to let each thread have their own instance, so the state store should be held as an instance variable on the Processor, alongside context, rather than as shared state on the supplier.

Also, suggest renaming `commandStateStore` to `store`.